tealiumlibrary/src/main/java/com/tealium/core/collection/AppCollector.kt

```diff
@@ -60,11 +61,12 @@ class AppCollector(private val context: Context, private val dataLayer: DataLaye
     }
 
     override suspend fun collect(): Map<String, Any> {
-        return mapOf(AppCollectorConstants.APP_RDNS to appRdns,
-                    AppCollectorConstants.APP_NAME to appName,
-                    AppCollectorConstants.APP_VERSION to appVersion,
-                    AppCollectorConstants.APP_BUILD to appBuild,
-                    AppCollectorConstants.APP_MEMORY_USAGE to appMemoryUsage)
+        return mapOf(
+            Dispatch.Keys.APP_RDNS to appRdns,
```

Comment:
Kind of picky, but also not big deal... This map isn't formatted and doesn't look cool like the rest.

Reply:
👍  - have run all of them through the auto-formatter now